### PR TITLE
Change the IPv4 subnet on Libvirt network

### PR DESCRIPTION
**Pull request by irosenzw**
Change the IPv4 subnet from 192.168.122.X
So it will be easier to run Hosted-engine setup
on a libvirt VM without change the local libvirt
network configuration.

Bug-Url: https://bugzilla.redhat.com/1698643
Signed-off-by: Ido Rosenzwig <irosenzw@redhat.com>

### CODE REVIEW

**Comment by irosenzw** [comment]
Verified on IPv4 network
@tiraboschi Can you please verify on IPv6 env as well ?

File: defaults/main.yml [hunk 1]
```diff
@@ -62,6 +62,12 @@ he_vm_mac_addr: null
 he_remove_appliance_rpm: true
 he_pki_renew_on_restore: false
 
+# If you decide to change these variables,
+# make sure they have the subnet mask
```
Comment (tiraboschi) [inline defaults/main.yml]: in default.xml we have:
  <ip address='192.168.122.1' netmask='255.255.255.0'>
but we are not editing the netmask attribute (and honestly I'm neither that sure it's worth).

File: defaults/main.yml [hunk 2]
```diff
@@ -62,6 +62,12 @@ he_vm_mac_addr: null
...
+he_local_vm_net_addr: "192.168.222.1"
+he_local_vm_dhcp_start: "192.168.222.2"
+he_local_vm_dhcp_end: "192.168.222.254"
+
```
Comment (tiraboschi) [inline defaults/main.yml]: For IPv6 we are just using he_ipv6_subnet_prefix.
I think we should do something analogue here:
he_ipv4_subnet_prefix: "192.168.222"
and then compute everything else starting from there and assuming a /24 subnet

File: defaults/main.yml (hunk 1 above)
Comment (irosenzw) [reply defaults/main.yml]: have what ?

File: defaults/main.yml (hunk 2 above)
Comment (irosenzw) [reply defaults/main.yml]: I thought about that, thought to give the user more flexibility playing with the ranges, but I'm guessing that's not that important.
I will change to he_ipv4_subnet_prefix

**Comment by irosenzw** [comment]
@tiraboschi please review

**Pull request merged**